src/cmd/test.ts

```diff
@@ -14,14 +14,15 @@ import { validateValues } from './validate-values'
 
 const cmdName = getFilename(import.meta.url)
 const tmpFile = '/tmp/otomi/test.yaml'
-const debug: OtomiDebugger = terminal(cmdName)
+let debug: OtomiDebugger = terminal(cmdName)
 
 const cleanup = (argv: Arguments): void => {
   if (argv.skipCleanup) return
-  unlinkSync(tmpFile)
+  if(existsSync(tmpFile)) unlinkSync(tmpFile)
 }
 
 const setup = (argv: Arguments): void => {
+  debug = terminal(cmdName)
```

Comment:
this is already done on line 17, so you can remove this line

Reply:
Brainfart sorry